doc/operators/ledger.rst

```diff
@@ -0,0 +1,53 @@
+Ledger
```

Comment:
There's already a `Ledger` page under `Developers/` (https://microsoft.github.io/CCF/master/developers/ledger.html), might be worth merging those two.

Reply:
Yeah I know, that's where the release notes point to now. It's a bit different though, it explains how the ledger is formed, it's more of a design document than a reference op doc.